src/components/Modals/Metadata/components/ConditionalLink/ConditionalLink.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { Anchor, Text } from 'grommet'
+
+export default function ConditionalLink({ color, text }) {
+  const expression = /[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)?/gi;
+  const regex = new RegExp(expression);
+  const isLink = text.match(regex)
+  return isLink ? <Anchor href={text} color='kelp' size='small'>{text}</Anchor>
+    : <Text color={color}>{text}</Text>
```

Comment:
This is a bit risky. This metadata is presumably user submitted content (from the researchers?) and we're setting it here to the href of an anchor, so vulnerable to XSS. I'd suggest planning on using markdown for the metadata values being rendered OR use something like dompurify to sanitize the string (https://github.com/cure53/DOMPurify).

With the classifier, we convert the string containing the URL to markdown: 

https://github.com/zooniverse/front-end-monorepo/blob/be8959d36c95d210e94cebaa01307283a215c2da/packages/lib-classifier/src/components/Classifier/components/MetaTools/components/Metadata/components/MetadataModal/MetadataModal.js#L29-L42

Then `Markdownz` handles sanitization via its remark-react dependency (defaults to github's sanitization schema: https://github.com/remarkjs/remark-react#optionssanitize and https://github.com/syntax-tree/hast-util-sanitize), however, we still test in the stories that it's working just to be sure (scroll down to the "Testing for vulerabilities" part):

https://zooniverse.github.io/front-end-monorepo/@zooniverse/react-components/index.html?path=/story/markdownz--light-theme-default